pkg/image/validation/kubernetes.go

```diff
@@ -134,3 +140,82 @@ func validateManifestURLs(k8s *image.Kubernetes) []FailedValidation {
 
 	return failures
 }
+
+func validateHelmCharts(k8s *image.Kubernetes, imageConfigDir string) []FailedValidation {
+	var failures []FailedValidation
+
+	if len(k8s.HelmCharts) == 0 {
+		return failures
+	}
+
+	seenHelmCharts := make(map[string]bool)
+	for _, chart := range k8s.HelmCharts {
+		if chart.Name == "" {
+			failures = append(failures, FailedValidation{
+				UserMessage: "Helm Chart 'name' field must be defined.",
+			})
+		}
+
+		if chart.Repo == "" {
+			failures = append(failures, FailedValidation{
+				UserMessage: "Helm Chart 'repo' field must be defined.",
+			})
+		} else if !strings.HasPrefix(chart.Repo, "http") && !strings.HasPrefix(chart.Repo, "oci://") {
+			failures = append(failures, FailedValidation{
+				UserMessage: "Helm Chart 'repo' field must begin with either 'oci://', 'http://', or 'https://'.",
+			})
+		}
+
+		if chart.Version == "" {
+			failures = append(failures, FailedValidation{
+				UserMessage: "Helm Chart 'version' field must be defined.",
+			})
+		}
+
+		if chart.CreateNamespace && chart.TargetNamespace == "" {
+			failures = append(failures, FailedValidation{
+				UserMessage: "Helm Chart 'createNamespace' field cannot be true without 'targetNamespace' being defined.",
+			})
+		}
+
+		if failure := validateHelmChartValues(chart.ValuesFile, imageConfigDir); failure != "" {
+			failures = append(failures, FailedValidation{
+				UserMessage: failure,
+			})
+		}
+
+		if _, exists := seenHelmCharts[chart.Name]; exists {
+			msg := fmt.Sprintf("The 'helmCharts' field contains duplicate entries: %s", chart.Name)
+			failures = append(failures, FailedValidation{
+				UserMessage: msg,
+			})
+		}
+
+		seenHelmCharts[chart.Name] = true
+	}
+
+	return failures
+}
+
+func validateHelmChartValues(valuesFile string, imageConfigDir string) string {
+	if valuesFile == "" {
+		return ""
+	}
+
+	if filepath.Ext(valuesFile) != ".yaml" && filepath.Ext(valuesFile) != ".yml" {
+		return "Helm Chart 'valuesFile' field must be the name of a valid yaml file ending in '.yaml' or '.yml'."
+	}
+
+	valuesFilePath := filepath.Join(imageConfigDir, "values", valuesFile)
```

Comment:
This should check for the file existing under `/helm/values`. Can we please export both of these directories from the combustion package and use them here?